src/install/dracut-install.c

```diff
@@ -489,8 +489,8 @@ static int resolve_deps(const char *src)
 {
         int ret = 0;
 
-        _cleanup_free_ char *buf = NULL;
         size_t linesize = LINE_MAX;
+        _cleanup_free_ char *buf = NULL;
```

Comment:
Uh. The commit is a complete mess now. Is there any reason to change order of declarations and add/remove empty lines?

Reply:
Yes, it looks better and I segragated code and variable declaration.

Reply:
Cleanup is fine. What my question was pointing to: is there any reason to do it *in the same commit as actual changes*?

Reply:
I get it. You want this to be in a different commit. I will see what I can do. Thanks.

Reply:
I have made it right. I will send a different PR with cleanup and consmetic changes if needed.
It will be too much to add this in this PR.